client/src/main/java/io/split/client/SplitClientConfig.java

```diff
@@ -150,6 +151,7 @@ private SplitClientConfig(String endpoint,
         _streamingServiceURL = streamingServiceURL;
         _telemetryURL = telemetryURL;
         _telemetryRefreshRate = telemetryRefreshRate;
+        _uniqueKeysTimeToSend = uniqueKeysTimeToSend;
```

Comment:
I recomen to use `uniqueKeysRefreshRate` to follow the convention.

Reply:
I changed the name.